src/pd/async/client.rs

```diff
@@ -191,15 +191,16 @@ impl PdClient for RpcClient {
                         leader: metapb::Peer,
                         down_peers: Vec<pdpb::PeerStats>,
                         pending_peers: Vec<metapb::Peer>,
-                        written_bytes: u64)
+                        region_stat: RegionStat)
```

Comment:
I prefer to put all fields like down_peers, pending_peers to the RegionStat like #1765 does.